.tekton/fetch-external-networks-oci-ta.yaml

```
@@ -0,0 +1,58 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: fetch-external-networks-oci-ta
+  namespace: rh-acs-tenant
+# TODO(ROX-23812): Refactor to a task bundle
+spec:
+  description: Fetches external network definitions from StackRox artifacts to be baked into the image.
+  params:
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the application source code with
+      the additional external networks definitions. This should be the result of the subscription-manager task,
+      to ensure order of execution.
```

[nit] `copy-subscription-manager-activation-key-oci-ta` and `fetch-external-networks-oci-ta` can be placed in any order to each other. The main thing is that they are sequential but that's a concern at the level of the pipeline. I'd suggest to remove it from description.

Also, similarly to above, `external network definitions` appear on `results` `SOURCE_ARTIFACT`.

```suggestion
  params:
  - name: SOURCE_ARTIFACT
    description: The Trusted Artifact URI pointing to the application source code.
```